docs/config.ld

```diff
@@ -21,6 +21,7 @@ topics={
     '02-contributing.md',
     '03-declarative-layout.md',
     '04-new-widgets.md',
+    '06-using-cairo.md',
     '90-FAQ.md',
```

Comment:
We should have this more spread out (to allow for new pages inbetween), since it appears to be used in links (which should be stable).